contao/templates/twig/content_element/_base.html.twig

```diff
@@ -0,0 +1,82 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% set sm = styleManager(data) %}
+{% set headline2 = data.headline2|deserialize %}
+{% set headlineClasses = sm.prepare('gHeadline').format(' %s') %}
+{% set insideClasses =
+    sm.prepare('cLayout', ['imageText']).format(' image_text') ~
+    sm.prepare('cLayout', ['iconText']).format(' icon_text') ~
+    sm.prepare('cLayout').format(' %s') ~
+    sm.prepare('eLayout').format(' %s') ~
+    sm.prepare('cText').format(' %s') ~
+    sm.prepare('eText').format(' %s') ~
+    sm.prepare('ePadding').format(' %s') ~
+    sm.prepare('cSpacing').format(' %s') ~
+    sm.prepare('cPadding').format(' %s') ~
+    sm.prepare('cBackground').format(' %s') ~
+    sm.prepare('eBackground').format(' %s')
+%}
+{% set sliderConfig =
+    sm.prepare('sliderConfig',['preset']).format(' data-slider-config=%s','json') ~
+    sm.prepare('slider').format(' data-slider=%s','json') ~
+    sm.prepare('sliderXS').format(' data-slider-xs=%s','json') ~
+    sm.prepare('sliderS').format(' data-slider-s=%s','json') ~
+    sm.prepare('sliderM').format(' data-slider-m=%s','json') ~
+    sm.prepare('sliderL').format(' data-slider-l=%s','json') ~
+    sm.prepare('sliderXL').format(' data-slider-xl=%s','json')
+%}
+{% set isListOrSlider = type != 'gallery' and (sm.get('eGridList') is not empty or 'init-slider' in element_css_classes) %}
+{% set isHeadline = type == 'headline' %}
+{% set contentHeadlineClasses = 'c_headline' ~ headlineClasses %}
+
+{# Add legacy classes for certain types #}
+{% set attributes = attrs()
+    .addClass('ce_headline', isHeadline)
+    .addClass('ce_image', type == 'image')
+    .addClass('ce_youtube', type == 'youtube')
+    .addClass('ce_vimeo', type == 'vimeo')
+    .mergeWith(attributes|default)
+%}
```

Comment:
Should we also add `ce_text` and the old float classes for embedded images within text-elements?

Reply:
As long as we support Contao 4.13 we should do this.

Reply:
See bc47da720da897f75be1ffb297514e45120cb35a